### ISSUE TITLE
Corruption of array passed to WriteMultipleRegistersAsync

### ISSUE THREAD

**Issue by tony-hailes** [Apollo3zehn/FluentModbus#52]
With the option Connect(port, ModbusEndianness.BigEndian) the byte order of each item in an integer array passed to WriteMultipleRegistersAsync is changed.

**Comment by tony-hailes** [comment]
In my application I use the array repeatedly, just changing one or two items in it. It means that before using the call I must make a copy of the array.

**Comment by Apollo3zehn** [comment]
Yes, that might happen because the array is rented from an ArrayPool. But your use case makes sense and there should be an option to pass your own array. I'll think about it.
